api/proto/v1/teleworker.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+option go_package = "internal/api/v1";
+
+package v1;
+
+service TeleWorker {
+  rpc Start(StartRequest) returns (StartResponse);
+  rpc Stop(StopRequest) returns (StopResponse);
+  rpc Status(StatusRequest) returns (StatusResponse);
+  rpc Stream(StreamRequest) returns (stream StreamResponse);
+}
+
+// UUID is used as a job identifier.
+message UUID {
+  string value = 1;
+}
+
+// StopSignal is a signal to be sent in order to stop a job.
+// TERM - signal to terminate the job. Yet, might fail to do that.
+// SIGINT - try to stop execution softly by requesting interruption. Might fail.
+// KILL - forcing the termination of the job. Terminates the job for sure.
+enum StopSignal {
+  TERM = 0;
+  SIGINT = 1;
+  KILL = 2;
+}
```

Comment:
You can keep it simple and just always send `SIGKILL`.

Reply:
Done, I've added a note to the doc and to the readme that we `SIGKILL` will always be used